HAAR-3531: import dev data from legacy auth

mpbostock opened the pull request:
Excluded the following as they had dual grant types and are not used explicitly anywhere in the `ministryofjustice` GitHub organisation and where they are used, the response from auth is mocked out:
```
test-client-one
interventions
book-a-prison-visit-client-ui
categorisationtool
```

AlTurner-MOJ reviewed (approved):
lgtm

AlTurner-MOJ reviewed (changes requested):
I think there's a few more tables that we may need to add data to as part of this? `oauth2_authorization_consent`, `oauth2_client_config` and `oauth2_client_deployment_details`

mpbostock commented:
> I think there's a few more tables that we may need to add data to as part of this? `oauth2_authorization_consent`, `oauth2_client_config` and `oauth2_client_deployment_details`

I've re-ran the migration and updated the other tables with the dev data. Can you have another look please @AlTurner-MOJ. I also tested the new UI for a client that didn't have any deployment details and the screens worked when I removed the entry with all nulls in.

AlTurner-MOJ reviewed (approved):
lgtm
